api/internal/managers/app/config/manager.go

```diff
@@ -23,6 +23,8 @@ type AppConfigManager interface {
 	TemplateConfig(configValues types.AppConfigValues, maskPasswords bool, filterHiddenItems bool) (types.AppConfig, error)
 	// GetConfigValues gets the current config values
 	GetConfigValues() (types.AppConfigValues, error)
+	// WarmEngineCache warms the template engine cache by executing templates with provided values
+	WarmEngineCache(configValues types.AppConfigValues) error
```

Comment:
can you explain why you need a new function and you cant just call `TemplateConfig(map[string]types.AppConfigValues{}, false, false)` or something like that to warm the cache?